src/main/java/com/es/phoneshop/model/product/SortField.java

```diff
@@ -0,0 +1,6 @@
+package com.es.phoneshop.model.product;
+
+public enum SortField {
+    description,
+    price
+}
```

Comment:
Here and in other places: constants in enums should follow constants code conventions and be upper case: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html